src/hammer-vlsi/hammer_vlsi/hammer_tool.py

```diff
@@ -28,6 +26,7 @@
 
 from .hammer_vlsi_impl import HierarchicalMode, LibraryFilter, HammerToolPauseException
 from .units import TimeValue, VoltageValue, TemperatureValue
+from .hammer_submit_command import HammerSubmitCommand, HammerBareSubmitCommand
```

Comment:
I don't think you need the `HammerBareSubmitCommand` import

Reply:
Correct, that was leftover from a previous commit. Fixed.